samples/async-context/worker.js

```diff
@@ -2,25 +2,26 @@
 // Licensed under the Apache 2.0 license found in the LICENSE file or at:
 //     https://opensource.org/licenses/Apache-2.0
 
-import { AsyncLocalStorage, AsyncResource } from 'node:async_hooks';
+import { AsyncLocalStorage, AsyncResource } from "node:async_hooks";
```

Comment:
Nit: I know it is a minor thing, but I generally have a style preference for single-quotes on JS string literals rather than double-quotes.

Reply:
We can switch that over and re-run prettier

Reply:
I'm also in favor of single quote